### PR TITLE
MBS-7674: add documents describing how to test WorkManager's Workers

### PR DESCRIPTION
**Pull request by int02h**

### CODE REVIEW

File: docs/content/test/WorkManager.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Work Manager Testing
+type: docs
+---
+
+# Work Manager Testing
+
+This documents describes how to test WorkManager's Workers.
```
Comment (eugene-krivobokov) [inline docs/content/test/WorkManager.md]: documents -> document

File: docs/content/test/WorkManager.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## WorkManager is disabled for tests
+
+Because using real WorkManager while testing involves a lot of problems as it starts to initialize 
+just before `AvitoTestApp` (using ContentProvider mechanism). Sometimes it leads to a classic 
+race condition: some Worker tries to get its dependencies to build its own Dagger graph but 
+Application isn't ready so test runner ends up with a crash.
+
+## How to test WorkManager's Workers?
```
Comment (eugene-krivobokov) [inline docs/content/test/WorkManager.md]: Let's move this explanation to the start. It's a real answer.

**Review by eugene-krivobokov — APPROVED** [review]

File: docs/content/test/WorkManager.md (hunk 1 above)
Comment (int02h) [reply docs/content/test/WorkManager.md]: fixed

File: docs/content/test/WorkManager.md (hunk 2 above)
Comment (int02h) [reply docs/content/test/WorkManager.md]: Agree. Move it closer to the beginning of the document.

File: docs/content/test/WorkManager.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+# Work Manager Testing
+
+This document describes how to test WorkManager's Workers.
+
+## How to test WorkManager's Workers?
+
+The best way to test your Workers is the way [described](https://developer.android.com/topic/libraries/architecture/workmanager/how-to/testing-210) 
+in official documentation. In short, you can test your Workers directly without initializing WorkManager.
+
+## WorkManager is disabled for tests
```
Comment (sergioboi) [inline docs/content/test/WorkManager.md:15]: we could add link to code disables it

Comment (int02h) [reply docs/content/test/WorkManager.md:15]: For what purpose?

**Pull request merged**